packages/ui/components/logo/Logo.tsx

```diff
@@ -1,18 +1,33 @@
 import classNames from "@calcom/lib/classNames";
-import { LOGO_ICON, LOGO } from "@calcom/lib/constants";
 
-export default function Logo({ small, icon }: { small?: boolean; icon?: boolean }) {
+export default function Logo({
+  small,
+  icon,
+  inline = true,
+  className,
+}: {
+  small?: boolean;
+  icon?: boolean;
+  inline?: boolean;
+  className?: string;
+}) {
   return (
-    <h3 className="logo inline ">
+    <h3 className={classNames("logo", inline && "inline", className)}>
       <strong>
         {icon ? (
-          <img className="mx-auto w-9 dark:invert" alt="Cal" title="Cal" src={LOGO_ICON} />
+          <img
+            className="mx-auto w-9 dark:invert"
+            alt="Cal"
+            title="Cal"
+            // TODO: Implement LOGO_ICON on endpoint
+            src="/api/logo?icon=1"
+          />
         ) : (
           <img
             className={classNames(small ? "h-4 w-auto" : "h-5 w-auto", "dark:invert")}
             alt="Cal"
             title="Cal"
-            src={LOGO}
+            src="/api/logo"
```

Comment:
Updated to use the previously made example. With this we don't even need a SubdomainProvider at all